sanic/config.py

```diff
@@ -6,10 +6,28 @@
 
 SANIC_PREFIX = "SANIC_"
 
+DEFAULT_CONFIG = {
+    "REQUEST_MAX_SIZE": 100000000,  # 100 megabytes
+    "REQUEST_BUFFER_QUEUE_SIZE": 100,
+    "REQUEST_TIMEOUT": 60,  # 60 seconds
+    "RESPONSE_TIMEOUT": 60,  # 60 seconds
+    "KEEP_ALIVE": True,
+    "KEEP_ALIVE_TIMEOUT": 5,  # 5 seconds
+    "WEBSOCKET_MAX_SIZE": 2 ** 20,  # 1 megabytes
+    "WEBSOCKET_MAX_QUEUE": 32,
+    "WEBSOCKET_READ_LIMIT": 2 ** 16,
+    "WEBSOCKET_WRITE_LIMIT": 2 ** 16,
+    "GRACEFUL_SHUTDOWN_TIMEOUT": 15.0,  # 15 sec
+    "ACCESS_LOG": True,
+}
+
 
 class Config(dict):
-    def __init__(self, defaults=None, load_env=True, keep_alive=True):
+    def __init__(self, defaults=None, load_env=True, keep_alive=None):
+        super().__init__(DEFAULT_CONFIG)
         super().__init__(defaults or {})
```

Comment:
Why run `super().__init__` twice? And not:

```initial = dict()
initial.update(DEFAULT_CONFIG)
initial.update(defaults or {})
super().__init__(initial)
```

Unless one is more performant than the other, I think it is a little more explicit what is going on in this version. I feel like it is not so common to see `super().__init__` called more than once in `__init__`.

Reply:
how about:
```python
def __init__(self, defaults=None, load_env=True, keep_alive=None):
    defaults = defaults or {}
    super().__init__({**DEFAULT_CONFIG, **defaults})
```
?

Reply:
Much cleaner :+1: